internal/scan-manager/scanners/veracode/src/main/java/org/wso2/security/tools/scanmanager/scanners/veracode/service/VeracodeScanner.java

```diff
@@ -47,9 +50,8 @@
 /**
  * Represents the Veracode Scanner.
  */
-@Component("VeracodeScannerImpl")
-@Scope(value = ConfigurableBeanFactory.SCOPE_SINGLETON)
-public class VeracodeScanner implements Scanner {
+@Component("VeracodeScannerImpl") @Scope(value = ConfigurableBeanFactory.SCOPE_SINGLETON) public class VeracodeScanner
```

Comment:
Is this the recommended style for annotations? Any reference? I feel like what we had earlier is much more readable

Reply:
Changed while formatting. Resolved now.